tests/phpt/dl/483_bcmath.php

```diff
@@ -121,3 +121,43 @@
         var_dump ("$u <=> $v = ".bccomp ($u, $v, $scale));
       }
 
+function test_bcmath_negative_zero() {
+/*
+  bcmath behaviour was fixed since 7.4.23 and 8.0.10 php versions.
+  In new php versions and in kphp bcmath never returns numbers like -0, -0.0 (but returns 0, 0.0).
+  See https://bugs.php.net/bug.php?id=78238 for more details.
+*/
+  #ifndef KPHP
+  if ((version_compare(PHP_VERSION, "7.4.23") >= 0) &&
+       (version_compare(PHP_VERSION, "8.0.0") < 0) ||
+       (version_compare(PHP_VERSION, "8.0.10") >= 0)) {
+  #endif
+    var_dump(bcadd("0", "-0.1"));    // 0
+    var_dump(bcadd("0", "-0.1", 1)); // -0.1
```

Comment:
Why do we need to keep tests returning non-zero here? If I'm not mistaken, their behavior must be the same in different php versions and kphp

Reply:
In order to test that code near is not broken.